ci-operator/step-registry/cucushift/hypershift-extended/disable-guest/cucushift-hypershift-extended-disable-guest-commands.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
```

Comment:
1. check where it is a hypershift hosted cluster env. otherwise stop steps.
2. add descriptions like assume ${SHARED_DIR}/mgmt_kubeconfig exists and it is magement cluster
3. check if file exists, otherwise the step stops